README.md

```diff
@@ -32,10 +32,20 @@ positional arguments:
 
 optional arguments:
   -h, --help         show this help message and exit
-  --address ADDRESS  Address for Flink JobManager
-  --port PORT        Port for Flink JobManager (default 8081)
+  --y                                If set, flink-scrat will use the {--app-name; --yarn-
+                                     address; --yarn-port} to try to find the correct yarn-
+                                     session info. Otherwise, it will assume that the
+                                     correct JobManager info is provided in {--address; --port}
+  --address ADDRESS                  Address for Flink JobManager
+  --yarn-address YARN_ADDRESS        Address for yarn manager
+  --port PORT                        Port for Flink JobManager (default 8081)
+  --yarn-port YARN_PORT              Port for yarn manager
 ```
 
+If `--y` is set, it is necessary to provide `--app-name`, `--yarn-address` and `--yarn-port`. Otherwise, `address` and `port` may be provided instead.
```

Comment:
I the following is more correct:`should be provided instead`

Reply:
👍